### PR TITLE
Remove code duplication in feedbackQuestionSubmissionEdit.jsp #3139

### LINKED ISSUE #3139: Remove code duplication in feedbackQuestionSubmissionEdit.jsp
Related to #3127

We should see if we can remove feedbackQuestionSubmissionEdit.jsp totally, and use  feedbackSubmissionEdit.jsp, instead of having duplicated code.

### PR DESCRIPTION
**Pull request by lejolly**
Fixes #3139 

The various conditional line breaks/spaces are to ensure that the outputs are exactly the same as before. 

The old file (feedbackQuestionSubmissionEdit.jsp) has not been deleted yet.

### CODE REVIEW

File: src/main/webapp/jsp/feedbackSubmissionEdit.jsp [hunk 1]
```diff
@@ -1,123 +1,131 @@
-<%@ page import="java.util.List"%>
-<%@ page import="teammates.common.util.TimeHelper"%>
-<%@ page import="teammates.common.util.Const"%>
+<%@ page import="java.util.List"%><%@ page import="java.util.ArrayList"%>
+<%@ page import="teammates.common.util.TimeHelper"%><%@ page import="teammates.common.util.Const"%>
```
Comment (LowWeiLin) [inline src/main/webapp/jsp/feedbackSubmissionEdit.jsp]: The imports should be on separate lines.

File: src/main/webapp/jsp/feedbackSubmissionEdit.jsp [hunk 2]
```diff
@@ -1,123 +1,131 @@
...
 <%@ page import="static teammates.ui.controller.PageData.sanitizeForHtml"%>
-
 <%
-	FeedbackSubmissionEditPageData data = (FeedbackSubmissionEditPageData)request.getAttribute("data");
-%>    
-    <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_SESSION_NAME%>" value="<%=data.bundle.feedbackSession.feedbackSessionName%>"/>
-    <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>" value="<%=data.bundle.feedbackSession.courseId%>"/>
-    <%
-    	if (data.account.googleId != null) {
+    boolean isQuestion = false;
```
Comment (LowWeiLin) [inline src/main/webapp/jsp/feedbackSubmissionEdit.jsp]: showSingleQuestion might be a clearer variable name.

File: src/main/webapp/jsp/feedbackSubmissionEdit.jsp [hunk 3]
```diff
@@ -1,123 +1,131 @@
...
-            <div class="row"><span class="help-block align-center"> 
-                <%=Const.FEEDBACK_SESSION_QUESTIONS_HIDDEN%>
-            </span></div>
-    <%    
         }
-    %>
+    %><% } %><%=isQuestion ? "\n    " : ""%>
+    <div class="well well-plain" <%=isQuestion ? "" : "id=\"course1\""%>><%=isQuestion ? "" : "\n            <div class=\"panel-body\">"%>
+        <%=isQuestion ? "" : "        "%><div class="form-horizontal"<%=isQuestion ? " role=\"form\"" : ""%>><%=isQuestion ? "" : "\n                    <div class=\"panel-heading\">"%>
+            <%=isQuestion ? "" : "            "%><div class="form-group">
```
Comment (LowWeiLin) [inline src/main/webapp/jsp/feedbackSubmissionEdit.jsp]: You don't have to add the condition on every single line... Look into what syntax JSP offers.

**Comment by LowWeiLin** [comment]
Highlighted some issues. Also, A cleaner way to do this without adding so much code into the jsp would be to filter out questions at the java level, and pass only the questions to be displayed into the jsp through the data bundle.

File: src/main/webapp/jsp/feedbackSubmissionEdit.jsp (hunk 3 above)
Comment (lejolly) [inline src/main/webapp/jsp/feedbackSubmissionEdit.jsp]: @LowWeiLin could you elaborate on the specific syntax? i'm not quite sure what else JSP offers, the only other syntax I could find was JSTL which isn't used here right?

**Comment by kanghj** [comment]
Closing this due to inactivity

**Pull request closed without merging**